docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/link/link.twig

```diff
@@ -36,18 +35,32 @@
   {% endset %}
 {% endif %}
 
+{% if is_external %}
+  {% set external_icon %}
+    {% include '@atoms/icon/icon.twig' with {
+       symbol: 'upper-right-arrow',
+     } only %}
+  {% endset %}
+{% endif %}
+
 {% set link_text %}
   {% if icon and button_icon %}
     {% if icon_placement == 'before' %}
-      {{ button_icon }}{{ text|raw }}
+      {{ button_icon }}{{ text|raw }} {{ external_icon }}
     {% elseif icon_placement == 'after' %}
-      {{ text|raw }}{{ button_icon }}
+      {{ text|raw }}{{ external_icon }}{{ button_icon }}
     {% endif %}
   {% else %}
-    {{ text|raw }}
+    {{ text|raw }}{{ external_icon }}
   {% endif %}
 {% endset %}
 
+{% if is_new_window %}
+  {% set link_text %}
+    {{ link_text }} <span class="civictheme-link--is-new-window">Opens in a new tab/window.</span>
```

Comment:
please rename to `civictheme-visually-hidden` and update `visually-hidden()` to `civictheme-visually-hidden()` mixin in `docroot/themes/contrib/civictheme/civictheme_library/components/00-base/visibility/_index.scss`